src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -24,6 +25,16 @@ def get(self, request, organization):
                     }
                 )
 
+        for provider_id in integrations_provider_bindings:
```

Comment:
can you do a `if features.has('organizations:internal-catchall', organization, actor=request.user):` before this whole block just to be safe?